src/components/molecules/ScheduleItemNG/ScheduleItemNG.tsx

```diff
@@ -36,6 +36,8 @@ import { RenderMarkdown } from "components/organisms/RenderMarkdown";
 
 import { ButtonNG } from "components/atoms/ButtonNG";
 
+import { getFirebaseStorageResizedImage } from "../../../utils/image";
```

Comment:
and more it to the "utils" import group.
```suggestion
import { getFirebaseStorageResizedImage } from "utils/image";
```

Reply:
https://github.com/sparkletown/sparkle/pull/2128/commits/8fc9e9becd300251bca5d4ca064d768167299581